examples/common/pigweed/rpc_services/Actions.h

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "actions_service/actions_service.rpc.pb.h"
+#include "app/util/attribute-storage.h"
+#include "pigweed/rpc_services/internal/StatusUtils.h"
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app/EventLogging.h>
+#include <platform/PlatformManager.h>
+
+namespace chip {
+namespace rpc {
+
+class Actions final : public pw_rpc::nanopb::Actions::Service<Actions>
```

Comment:
Please describe a bit more the usage of this debug interface. Notably, it has up to 3 "generic" arguments per command, but this fact is not clearly documented.

Reply:
Done